<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createMemberdialogue</class>
 <widget class="QDialog" name="createMemberdialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>54</y>
     <width>47</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="membershipLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>26</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Membership Number:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>52</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="execLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>76</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="expireLabel">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>106</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Experation Date:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="expireDate">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>104</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="memberIDSpinbox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
  <widget class="QComboBox" name="typeComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>76</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Executive</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Regular</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>createMemberdialogue</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>createMemberdialogue</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
